documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -181,6 +181,18 @@ requirements:
 * Has a ``get_buffer`` method with no arguments. The method returns the SYCL buffer passed to
   ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` functions
 
+Also ``begin``, ``end`` can optionally accept SYCL deduction tags and ``sycl::no_init``
+(See SYCL 2020: 4.7.6.3, 4.7.6.4 for details) to explicitly mention which access mode should be applied
+to the buffer accessor when submitting DPC++ kernel to a device, e.g.
+
+.. code:: cpp
+
+  auto first1 = begin(buf, sycl::read_only);
```

Comment:
Probably add some comment in the code like ``auto first1 = begin(buf, sycl::read_only); // buffer<...>`` with concrete template arguments. Otherwise, it might be unclear to the user what is the result type.
I would prefer to have such a comment for each line

Reply:
Seems it's unnecessary now.